common/alert/alerts-en.html

```diff
@@ -0,0 +1,154 @@
+</section>
+<section class="alert alert-info">
+	<h3>Info alert</h3>
+	<p>Alert details.</p>
+</section>
+<h3>Code</h3>
+<pre><code>
+	&lt;section class="alert alert-danger"&gt;
+		&lt;h3&gt;Danger alert&lt;/h3&gt;
+		&lt;p&gt;Alert details.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section class="alert alert-warning"&gt;
+		&lt;h3&gt;Warning alert&lt;/h3&gt;
+		&lt;p&gt;Alert details.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section class="alert alert-success"&gt;
+		&lt;h3&gt;Success alert&lt;/h3&gt;
+		&lt;p&gt;Alert details.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section class="alert alert-info"&gt;
+		&lt;h3&gt;Info alert&lt;/h3&gt;
+		&lt;p&gt;Alert details.&lt;/p&gt;
+	&lt;/section&gt;</code></pre>
+
+<h2>links inside Alerts</h2>
+<section class="alert alert-success">
+	<h3>(Success Title)</h3>
+	<p>(Success content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-info">
+	<h3>(Info Title)</h3>
+	<p>(Info content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-warning">
+	<h3>(Warning Title)</h3>
+	<p>(Warning content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-danger">
+	<h3>(Danger Title)</h3>
+	<p>(Danger content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<h3>Code</h3>
+<pre><code>
+	&lt;section <strong>class=&quot;alert alert-success&quot;</strong>&gt;
+	&lt;p&gt; Success content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section <strong>class=&quot;alert alert-info&quot;</strong>&gt;
+	&lt;p&gt; Info content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section <strong>class=&quot;alert alert-warning&quot;</strong>&gt;
+	&lt;p&gt; Warning content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
+	&lt;/section&gt;
+
+	&lt;section <strong>class=&quot;alert alert-danger&quot;</strong>&gt;
+	&lt;p&gt; Danger content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
+	&lt;/section&gt;
+</code></pre>
+
+<h2> Alerts without Headings <h2>
+<h3> Success alert </h3>
+<div class="alert alert-success">
+	<p>(Success content here.)</p>
+</div>
+
+<h3> Info alert </h3>
+<div class="alert alert-info">
+	<p> Info content here. </p>
+</div>
+
+<h3> Warning alert </h3>
+<div class="alert alert-warning">
+	<p>(Warning content here.)</p>
+</div>
+
+<h3> Danger alert </h3>
+<div class="alert alert-danger">
+	<p>(Danger content here.)</p>
+</div>
+
+<h3>Code</h3>
+<pre><code>
+	//Sucess Alert with a heading
+	&lt;section <strong>class=&quot;alert alert-success&quot;</strong>&gt;
+	<strong>  &lt;h*&gt;...&lt;/h*&gt;</strong>
+	...
+	&lt;/section&gt;
+
```

Comment:
Remove this code sample as it don't match the immediate preceding working example "Alerts without Headings"

Reply:
The requested change was applied